### ISSUE TITLE
where is logstash?

### ISSUE THREAD

**Issue by ghost** [DanielBerman/ansible-elk-playbook#1]
I tried this playbook a couple of times.  It runs to completion without errors.  

When I get on the server, there's no logstash.  Thoughts?

root@ip-10-0-1-170:~# locate logstash
/usr/share/kibana/src/ui/public/icons/logstash-color.svg
/usr/share/kibana/src/ui/public/icons/logstash-gray.svg
/usr/share/kibana/src/ui/public/icons/logstash.svg
root@ip-10-0-1-170:~# 

(yes I ran updatedb before using locate)

**Comment by ritwik12** [comment]
@hullsean Yes, I noticed too. There is no logstash. I have created role for logstash in my clone. You can have it here https://github.com/ritwik12/ansible-playbook-ELK/tree/master/roles/logstash/tasks

**Comment by ghost** [comment]
Thx Ritwik
